src/main/java/org/roaringbitmap/longlong/MutableRoaringBitmapSupplier.java

```diff
@@ -0,0 +1,14 @@
+package org.roaringbitmap.longlong;
+
+import org.roaringbitmap.BitmapDataProvider;
+import org.roaringbitmap.buffer.MutableRoaringBitmap;
+
+public class MutableRoaringBitmapSupplier implements BitmapDataProviderSupplier {
+  private static final long serialVersionUID = -1332258289943674375L;
```

Comment:
I don't think this is necessary as the class is never serialized.

Reply:
Fixed. Thanks